LocalSettings.php

```diff
@@ -3146,8 +3147,12 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'read' => true,
+				'checkuser' => true,
```

Comment:
Unfortunately, we currently don't give these permissions to users on wikis, they are only provided for stewards.